.github/workflows/postTestsResults.js

```diff
@@ -0,0 +1,118 @@
+const { Command } = require("commander");
+const program = new Command();
+const { Octokit } = require("@octokit/core");
+
+program
+  .name("Send tests results")
+  .description(
+    "Get tests results from testmo and post message to slack or on release PR",
+  )
+  .option("--run_id <run_id>", "Testmo run id")
+  .option(
+    "--testmo_token <testmo_token>",
+    "Bearer token for authorization in testmo",
+  )
+  .option(
+    "--slack_webhook_url <slack_webhook_url>",
+    "Should send notification on slack",
+  )
+  .option("--environment <environment>", "Environment")
+  .option("--url_to_action <url_to_action>", "Url to enter github action")
+  .option("--ref_name <ref_name>", "Ref to point where tests where run")
+  .action(async options => {
+    const runId = options.run_id;
+    const testmoAuthToken = options.testmo_token;
+    const testsResults = await getTestsStatus(runId, testmoAuthToken);
+    const testsStatus = convertResults(
+      testsResults,
+      options.environment,
+      options.ref_name,
+    );
+
+    await sendMessageOnSlack(
+      testsStatus,
+      options.slack_webhook_url,
+      options.url_to_action,
+    );
+  })
+  .parse();
+
+async function getTestsStatus(runId, testmoToken) {
+  const runResult = await fetch(
+    `https://saleor.testmo.net/api/v1/automation/runs/${runId}`,
+    {
+      headers: {
+        Authorization: `Bearer ${testmoToken}`,
+      },
+    },
+  );
+  return await runResult.json();
+}
+
+function convertResults(results, environment, refName) {
+  let status = results?.result?.status === 2 ? "SUCCESS" : "FAILURE";
+  let message = `Tests run on environment: \n${environment}\n`;
+  const linkToResults = `https:\/\/saleor.testmo.net\/automation\/runs\/view\/${results.result.id}`;
```

Comment:
I don't think we need backlashes? 
```suggestion
  const linkToResults = `https://saleor.testmo.net/automation/runs/view/${results.result.id}`;
```